shotgun_api3/shotgun.py

```diff
@@ -2014,6 +2014,12 @@ def schema_field_update(self, entity_type, field_name, properties):
         :param field_name: Internal Shotgun name of the field to update.
         :param properties: Dictionary with key/value pairs where the key is the property to be
             updated and the value is the new value.
+        :param dict project_entity: Optional Project entity specifying which project to modify the
+            ``visible`` property for. If the ``visible`` is present in ``properties`` and
+            ``project_entity`` is not set, an exception will be raised. Example:
+            ``{'type': 'Project', 'id': 3}``
+        :param project_entity: Link to the project we wish to change the field visibility for.
```

Comment:
Looks like you've documented `project_entity` twice here.

Reply:
Also, does `project_entity` only apply for `visible` property?  Is that something we want to call out more explicitly?

Reply:
Good point. Let me know what you think of my update.